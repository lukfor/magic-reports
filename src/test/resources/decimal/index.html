{{decimal(100000)}}
{{decimal(integer)}}
{{decimal(long)}}
{{decimal(double)}}
{{decimal(float)}}
{{decimal(string)}}
{{decimal("1")}}