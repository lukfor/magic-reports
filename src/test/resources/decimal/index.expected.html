100,000
1,000,000
454,444,444
0
1
NaN
NaN